test/controllers/OtherReasonController.test.ts

```diff
@@ -45,10 +51,17 @@ describe('OtherReasonController', () => {
     describe('POST request', () => {
         it('should return 422 response with rendered error messages when invalid data was submitted', async () => {
 
-            const app = createApp({});
+            const appeal = {
+                penaltyIdentifier: {
+                    companyNumber: 'NI000000',
+                    penaltyReference: 'A00000001'
+                }
+            } as Appeal;
+
+            const app = createApp({ appeal });
 
             await request(app).post(OTHER_REASON_PAGE_URI)
-                .send({})
+                .send(appeal)
```

Comment:
That seems wrong. Appeal isn't submitted when POST request is made on other reason page.